be/src/olap/push_handler.cpp

```diff
@@ -382,6 +405,52 @@ void PushBrokerReader::print_profile() {
     LOG(INFO) << ss.str();
 }
 
+Status PushBrokerReader::_get_next_reader() {
+    while (true) {
+        _cur_reader.reset(nullptr);
```

Comment:
if cur_reader != nullptr, should close it first and then set it to null?

Reply:
cur_reader will in destructor to call close().